ksy_schema.json

```diff
@@ -82,6 +82,151 @@
         "xref": {
           "type": "object",
           "additionalProperties": false,
+          "properties": {
+            "forensicswiki": {
+              "description": "article name at [Forensics Wiki](https://forensicswiki.org/wiki/Main_Page), which is a CC-BY-SA-licensed wiki with information on digital forensics, file formats and tools\n\nfull link name could be generated as `https://forensicswiki.org/wiki/` + this value",
```

Comment:
This should be fixed in the original documentation as well, but https://forensicswiki.org/ went down a few months ago, and the site is apparently located at https://forensicswiki.xyz/ now.

Reply:
@dgelessus Thanks for the info, I didn't know that. I'll fix it.